js/apps/system/_admin/aardvark/APP/frontend/js/models/arangoCollectionModel.js

```diff
@@ -289,6 +302,15 @@
         waitForSync: wfs
       };
 
+      if (cacheEnabled === 'true') {
+        cacheEnabled = true;
+      } else if (cacheEnabled === 'false') {
+        cacheEnabled = false;
+      }
+      if (cacheEnabled !== null && cacheEnabled !== undefined) {
+        data.cacheEnabled = cacheEnabled;
+      }
```

Comment:
If `cacheEnabled` is any string other than `'true'` or `'false'`, it still gets set in `data.cacheEnabled`, as a string. Either that is not a problem or we must be sure `cacheEnabled` is never any value other than `true|false|null|undefined`.

Personally, I'd prefer a if `cacheEnabled` defaulted to `false` for any _falsey_ value, and `true` otherwise. So Something like
`data.cacheEnabled = !!cacheEnabled;`

Reply:
I don't think that `!!cachEnabled` would be correct, given the following experiment:
```
[true, false, "true", "false", null, undefined, 1, 0, "piff"].forEach((value) => { print("value: ", value, "bool: ", !!value); });
value:  true bool:  true
value:  false bool:  false
value:  true bool:  true
value:  false bool:  true
value:  null bool:  false
value:  undefined bool:  false
value:  1 bool:  true
value:  0 bool:  false
value:  piff bool:  true
```

Reply:
Good catch, thanks!